reference/src/main/scala/akka/stream/alpakka/reference/impl/ReferenceFlow.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.reference.impl
+
+import akka.annotation.InternalApi
+import akka.event.Logging
+import akka.stream.{Attributes, FlowShape, Inlet, Outlet}
+import akka.stream.alpakka.reference.ReferenceWriteMessage
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+
+/**
+ * private package hides the class from the API in Scala. However it is still
+ * visible in Java. Use "InternalApi" annotation to communicate to Java developers
+ * that this is private API.
```

Comment:
No `INTERNAL API` doc comment? Is the annotation visible in JavaDoc?

Reply:
Ahh, forgot that. We do not generate javadoc so I forgot to check.